osmaxx/production_example.yml

```diff
@@ -0,0 +1,37 @@
+version: '2'
+services:
+  osmaxx-proxy:
+    environment:
+      # comma separated list, no brackets, e.g. localhost,dev.myhost.com
+      - VIRTUAL_HOST=<my.domain.org>
+      - LETSENCRYPT_HOST=<my.domain.org>
+      - LETSENCRYPT_EMAIL=<email@my.domain.org>
+  frontend:
+    environment:
+      # comma separated list, no brackets, e.g. localhost,dev.myhost.com
+      - DJANGO_ALLOWED_HOSTS=<my.domain.org>
+      - SENTRY_DSN=
```

Comment:
I couldn't decide if an example value would be better, or just leave it empty o make it clear, that this needs to be filled in.